.github/workflows/build.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   build-and-push-image:
     if: "!contains(github.event.head_commit.message, '[maven-release-plugin]')"
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```

Comment:
Should we go with latest or check which one is currently latest and use that one?

Reply:
I went with it intentionally, it is going to serve as a safety net in case that we meet similar issues in the future.